Formula/jolie.rb

```diff
@@ -7,15 +7,9 @@ class Jolie < Formula
   depends_on "openjdk"
 
   def install
-    system "java", "-jar", "jolie-#{version}.jar", "--jolie-home", libexec, "--jolie-launchers", include
-    Dir.each_child(include) do |filename|
-      file = bin/filename
-      file.write <<~EOS
-        #!/bin/sh
-        export JOLIE_HOME=#{libexec}
-        sh #{include}/#{filename} $@
-      EOS
-    end
+    system "java", "-jar", "jolie-#{version}.jar", "--jolie-home", libexec, "--jolie-launchers", libexec/"bin"
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files libexec/"bin", :JOLIE_HOME => "${JOLIE_HOME:-#{libexec}}"
```

Comment:
if these run in java you probably also want to add the `JAVA_HOME` from the formula I linked

Reply:
Thanks for the heads-up, included